clash-lib/src/Clash/Rewrite/Util.hs

```diff
@@ -199,12 +200,11 @@ runRewriteSession :: RewriteEnv
                   -> RewriteState extra
                   -> RewriteMonad extra a
                   -> a
-runRewriteSession r s m = traceIf True ("Clash: Applied " ++
-                                        show (s' ^. transformCounter) ++
-                                        " transformations")
-                                  a
-  where
-    (a,s',_) = runR m r s
+runRewriteSession r s m = unsafePerformIO $ do
+  let (a,s',_) = runR m r s
+  encodeFile "history.dat" (s' ^. rewriteSteps)
```

Comment:
Is this supposed to be in this commit?

Reply:
Oops, you're right. Forgot to save the file when editing.